ide/scam-alerts.json

```diff
@@ -2,6 +2,7 @@
   "alerts" : [
     { "message" : "The only URL Remix uses is remix.ethereum.org", "url" : "" },
     { "message" : "Beware of online videos promoting liquidity front runner bots", "url": "https://medium.com/remix-ide/remix-in-youtube-crypto-scams-71c338da32d" },
```

Comment:
Please update "Beware of online videos promoting liquidity front runner bots" to:
Beware of videos promoting MEV, crypto loans, or trading bots.